### PR TITLE
[WIP] [DX] Add RectorConfig::onlyRule() for faster upgrades

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by staabm** [comment]
Wdyt about having a cli arg which enables only a single rule and ignores all other rules/rule-sets?

That way one can easily trigger the migration without the need to fiddle with the config file

**Comment by samsonasik** [comment]
@staabm the rule execute can be configurable, pass via cli arg will make it harder to set as require more args to implode

File: packages/Config/RectorConfig.php [hunk 1]
```diff
@@ -227,6 +227,24 @@ public function rule(string $rectorClass): void
         SimpleParameterProvider::addParameter(Option::REGISTERED_RECTOR_RULES, $rectorClass);
     }
 
+    /**
+     * @api
+     * @internal use only for step-by-step upgrade, runs faster and only with this one rule loaded
+     */
+    public function onlyRule(string $rectorClass): void
```
Comment (samsonasik) [inline packages/Config/RectorConfig.php:234]: How about add `onlyRuleWithConfiguration()` as well?

**Comment by TomasVotruba** [comment]
I'm thinking how to disable sets loading.... any ideas?

**Comment by TomasVotruba** [comment]
Closing for now, as now I see little added value over simply using custom rector config.

**Pull request closed without merging**